### PR TITLE
Issue1192 revision bestest hp

### PR DESCRIPTION
**Pull request by JavierArroyoBastida**
This is for https://github.com/open-ideas/IDEAS/issues/1192.

### CODE REVIEW

File: IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo
Comment (Mathadon) [inline IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo:25]: We should see how to best set the value of the parameter `lInt`. This is the internal wall length. By default it equals the length of side A but it should be longer. If we assume that each room within the zone is 4x4 meters then we have 12 rooms. Only the outer walls of those 12 rooms are implemented so we need `lInt = 3 x w + 2 x l` for representing the internal wall mass. @JavierArroyoBastida @dhblum do you agree with this reasoning?

@JavierArroyoBastida if you implement the change, it would be interesting to check how the simulation results change.

**Review by Mathadon — CHANGES_REQUESTED** [review]

File: IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo
Comment (JavierArroyoBastida) [reply IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo:25]: Sounds reasonable, I'll implement the edit and show the simulation results change here.

Comment (dhblum) [reply IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo:25]: Sounds reasonable to me.

Comment (JavierArroyoBastida) [reply IDEAS/Examples/IBPSA/SingleZoneResidentialHydronicHeatPump.mo:25]: I've increased internal wall mass here: https://github.com/open-ideas/IDEAS/pull/1193/commits/87620802dfe39a607f11bf312d775bc735748cfc when assuming 12 rooms. 
Below you can see the simulation results of the operative temperature, supply water temperature and HP modulating signal. A comparison is made between the original implementation, i.e. original size and without any internal wall (original is simulation 1 and modified is simulation 3).  
image.

The effects of reducing size and adding internal walls sort of compensate each other regarding thermal mass, although they also increase supply water temperature to the radiant floor. I've increased the offset used in control (in here https://github.com/open-ideas/IDEAS/pull/1193/commits/c14701161679ebd5f6a8c06330395a527d9c34f7) to avoid higher supply water temperatures. See below simulation results (simulation 3) again against the very original implementation (simulation 2). 

image

**Comment by JavierArroyoBastida** [comment]
@Mathadon please feel free to review again but do not merge until we decide on whether we change the pricing scenarios as well since that'd require also edits on the documentation of this model.

**Comment by Mathadon** [comment]
I’ll wait then since I have to run unit tests on my PC now that travis no longer gives (a lot of) free credit.

**Comment by JavierArroyoBastida** [comment]
@Mathadon this is ready for review. Corresponding BOPTEST edits will come later in https://github.com/ibpsa/project1-boptest/issues/289 once this is merged. Notice that this also updates pricing profiles by adding constant transmission fees and taxes to pricing scenarios. Just FYI below you can see how price profiles change for electricity: 
image

**Pull request merged**
